### PR TITLE
Adding script to install Node.js

### PR DESCRIPTION
**Pull request by garnold**

### CODE REVIEW

File: scripts/configure-noip-client [hunk 1]
```diff
@@ -10,7 +10,7 @@ fi
 
 pushd /usr/local/src/
 
-/usr/bin/sudo /usr/bin/wget --quiet 'http://www.no-ip.com/client/linux/noip-duc-linux.tar.gz'
+/usr/bin/sudo /usr/bin/curl --location --remote-name --silent 'http://www.no-ip.com/client/linux/noip-duc-linux.tar.gz'
```
Comment (pgroudas) [inline scripts/configure-noip-client:13]: is there an advantage of curl vs wget?

File: scripts/install-bluez-5 [hunk 2]
```diff
@@ -10,12 +10,12 @@ fi
...
 pushd /usr/local/src
 
-/usr/bin/sudo /usr/bin/wget --quiet "http://www.kernel.org/pub/linux/bluetooth/bluez-${BLUEZ_VERSION}.tar.xz"
+/usr/bin/sudo /usr/bin/curl --location --remote-name --silent "http://www.kernel.org/pub/linux/bluetooth/bluez-${BLUEZ_VERSION}.tar.xz"
 /usr/bin/sudo /bin/tar xf "bluez-${BLUEZ_VERSION}.tar.xz"
 
 pushd "bluez-${BLUEZ_VERSION}"
 
-/usr/bin/sudo ./configure --disable-systemd
+/usr/bin/sudo ./configure --disable-systemd --enable-library
```
Comment (pgroudas) [inline scripts/install-bluez-5:18]: Is this the change to how we install bluez?

File: scripts/install-nodejs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+set -e
+# set -x
+
+export DEBIAN_FRONTEND=noninteractive
+
+pushd /tmp/
+
+# http://node-arm.herokuapp.com/
+/usr/bin/curl --location --remote-name --silent 'http://node-arm.herokuapp.com/node_latest_armhf.deb'
```
Comment (pgroudas) [inline scripts/install-nodejs:11]: Kinda annoying that latest isn't a reference to a static version, but whaddaya gonna do?

File: scripts/install-bluez-5 (hunk 2 above)
Comment (garnold) [inline scripts/install-bluez-5:18]: Yessir.  The switch installs the lib and header files required for `noble` / `xpc-connection` to compile.

File: scripts/configure-noip-client (hunk 1 above)
Comment (garnold) [inline scripts/configure-noip-client:13]: `wget` doesn't overwrite the file if it already exists (it creates a `.0` file instead), whereas `curl` will overwrite it.

File: scripts/install-nodejs (hunk 3 above)
Comment (garnold) [inline scripts/install-nodejs:11]: My thoughts exactly :)

**Pull request merged**